scripts/generate-status-pages.ts

```diff
@@ -0,0 +1,54 @@
+import fs from "node:fs/promises";
+import path from "node:path";
+
+async function generateStatusPages() {
+  const { domainToStatusPageUrls, ...restStatusPages } = await import(
+    "@sentry/status-page-list"
+  );
+
+  const statusPages = Object.values(restStatusPages).sort((value) =>
+    value.name.localeCompare(value.name)
+  );
+
+  console.log(
+    `Found ${statusPages.length} status pages from @sentry/status-page-list`
+  );
+
+  const statusPagesMarkdown = statusPages
+    .map(
+      ({ name, statusPageUrl }) =>
+        `* [${name}](${statusPageUrl}) - ${name} status page`
```

Comment:
This means we lose customised descriptions. Arguably we could [upstream a description to Sentry](https://github.com/getsentry/status-page-list/blob/main/src/types.ts#L1)

Reply:
I'd be open to add descriptions to the sentry library